it/uk/gov/hmrc/partnershipidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -94,11 +94,28 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
 
```

Comment:
We have tests for incorporated partnerships when the Sa postcode check fails, but there are no tests for when the Sa postcode test passes but the company postcode does not.